components/Radio/src/stories/bem.stories.mdx

```diff
@@ -54,9 +54,15 @@ import readme from "../../README.md";
   <Story name="Without label">
     {() => (
```

Comment:
Volgens mij mag de react uit de bem, hiervoor is een aparte react.stories.mdx file